sign-in.txt

```diff
@@ -2,7 +2,9 @@
 SIGN IN SHEET FOR GITFLOW
 Done When: Your LAST name and github username are listed below the last name on the list below 
 
- LAST, First              GitHub USERNAME
-  ______________      | ___________________
-  Bansenauer, Brian   |   Bansenauer-Cascadia
-  Fellstrom,  Chase   |    Chasefells
+ LAST, First                 GitHub USERNAME
+  ______________        |   ___________________
+  Bansenauer, Brian     |   Bansenauer-Cascadia
+  Fellstrom,  Chase     |   Chasefells
+  Schildwachter, Adrien |   AdrienSch
```

Comment:
HEy this looks good, but I want to wait for other pull requests so we can practice merge conflict resoluti0n